app/services/katello/pulp3/repository.rb

```diff
@@ -497,7 +497,8 @@ def add_content(content_unit_href, remove_all_units = false)
       rescue api.client_module::ApiError => e
         if e.message.include? 'Could not find the following content units'
           raise ::Katello::Errors::Pulp3Error, "Content units that do not exist in Pulp were requested to be copied."\
-            " Please run a complete sync on the following repository: #{repository_reference.root_repository.name}. Original error: #{e.message}"
+             " Please run `foreman-rake katello:delete_orphaned_content` to fix the following repository:"\
+            " #{::Katello::Pulp3::RepositoryReference.find_by(repository_href: repository_reference.repository_href).root_repository.name}. Original error: #{e.message}"
```

Comment:
```suggestion
            " #{repository_reference.root_repository.name}. Original error: #{e.message}"
```

Reply:
The `repository_reference` can be used here since the method is adding content to the service class's own repo.